src/main/java/core/basesyntax/ColorList.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum ColorList {
```

Comment:
```suggestion
public enum Color {
```

в названиях enum не нужно добавлять слово List